src/services/AudiusSubgraph/index.ts

```diff
@@ -0,0 +1,54 @@
+import { useQuery, gql } from '@apollo/client'
+
+const HEALTH_CHECK = gql`
+  query healthCheck($subgraphName: String!) {
+    indexingStatusForCurrentVersion(subgraphName: $subgraphName) {
+      synced
+      health
+      fatalError {
+        message
+        block {
+          number
+          hash
+        }
+        handler
+      }
+      chains {
+        chainHeadBlock {
+          number
+        }
+        latestBlock {
+          number
+        }
+      }
+    }
+  }
+`
+
+type Health = 'healthy' | 'unhealthy' | 'failed'
+interface HealthCheck {
+  healthCheck: {
```

Comment:
man I wish we could autogen these from the gql queries... someday

Reply:
Oh, I didn't mean to commit this, but should be fine if it's in the bundle though. 👀 
This isn't used... yet but can be used to check the health of the subgraph and if it's caught up.